content/updates/2017-06-12-release-notes.md

```diff
@@ -0,0 +1,29 @@
++++
+date = "2017-06-12"
+title = "Platform Release Notes: June 12, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Changed
+
+- [Static IP addresses for communicating with external apps](https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses): Outbound traffic from cloud.gov now comes from specific IP addresses to help customers open up a connection between cloud.gov and outside data centers.
```

Comment:
This anchor doesn't link to anything on https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses - seems ok to just remove the anchor.

Also good to use relative links for links within the site - https://github.com/18f/cg-site#links - not required, but nice for future-proofing.